httpie/output/formatters/xml.py

```diff
@@ -26,9 +40,18 @@ def pretty_xml(document: 'Document',
     if standalone is not None and sys.version_info >= (3, 9):
```

Comment:
Why do we deal with `standalone`? I think it should have no impact on our use case, and we should treat all XML docs as standalone (to avoid any potential loading of linked external stuff).

Reply:
I have not touched/interacted with this part, and I don't really know. We pass it depending on whether the initial declaration is `standalone` or not.

Reply:
> > I think we should also include a test that verifies the behavior we’re correcting (i.e., that an unwanted XML declaration is indeed added by the library).
> 
> We already have such tests (we had tests that did not have a declaration in their raw version, but have it on their formatted. So I simply fixed them).

What I mean is that we implicitly rely on the library auto-adding the declaration. For example, we assume there’ll always be `lines[0]`. 

Let’s add an explicit test case verifying this assumption still holds (something like `assert pretty('<x/>') == '<?xml …?><x/>'`).

 
> explicitly mention about the formatters are only support for semantical equivalence.

Let’s do that.

Reply:
Could you please still clean it up even though it’s older code?

Reply:
Sure thing.